packages/svelte/tests/runtime-legacy/samples/whitespace-bom/main.svelte

```diff
@@ -0,0 +1 @@
+<div>&#xFEFF;</div>
```

Comment:
BOM characters can only appear at the very beginning of a file. I don't think we need the logic as presented here, we should just look at the very first charcode and remove it if it's a BOM

Reply:
I tried that but I couldn't get a test case that emulated it. :/ Feel free to try if you have an approach that can work.

Reply:
But isn't it a bug if someone has a BOM character not at the start? Not sure we should handle that / slow down our parser for finding these things. Also, AFAIK a BOM character can also be the start of a valid character sequence that means something completely different.

I think it's ok to not have a test for this, as long as there's a small comment explaining what the code does.

Reply:
Changed the PR to strip BOM once at the start, added a test

Reply:
Looks good!